nodejs/eks/nodegroup.ts

```diff
@@ -288,9 +289,6 @@ export function createNodeGroup(name: string, args: NodeGroupOptions, parent: pu
     if (core.vpcCni !== undefined) {
         cfnStackDeps.push(core.vpcCni);
     }
-    if (core.eksNodeAccess !== undefined) {
-        cfnStackDeps.push(core.eksNodeAccess);
-    }
```

Comment:
Why is this change made?

Reply:
`eksNodeAccess` has been replaced with `kubectl apply`, and it was being stored in the `createCore` output `CoreData`. Since we no longer use the CM this dependency is irrelevant and undefined.